pom.xml

```diff
@@ -42,26 +42,34 @@ THE SOFTWARE.
       <comments>All source code is under the MIT license.</comments>
     </license>
   </licenses>
-
+    
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <compileSource>1.7</compileSource>
+    <compileTarget>1.7</compileTarget>
+  </properties>
+    
   <build>
     <pluginManagement>
       <plugins>
         <plugin>
-          <groupId>org.apache.maven.plugins</groupId>
-          <artifactId>maven-enforcer-plugin</artifactId>
-          <version>1.0-beta-1</version>
+          <artifactId>maven-clean-plugin</artifactId>
+          <version>${maven-clean-plugin.version}</version>
+        </plugin>
+        <plugin>
+          <artifactId>maven-compiler-plugin</artifactId>
+          <version>${maven-compiler-plugin.version}</version>
+          <configuration>
+            <source>${compileSource}</source>
+            <target>${compileTarget}}</target>
+            <showDeprecation>false</showDeprecation>
+            <showWarnings>false</showWarnings>
+          </configuration>
```

Comment:
Hmm... Somehow the comments are outdated.

Those configurations look the default behavior and I think they don't affect the actual behavior.
I don't know so much about maven, and I expect a review by another developer good at maven.